### DISCUSSION TITLE
new version extra files?

### DISCUSSION THREAD

**Discussion by Kerie70** [zippity21/Stardew_Valley_Regression_Mod#68] [General]
Was the new download supposed to come with regression 1.2.1 and 1.2.2 zip files under the regression folder or was it an accident and I can just delete them? 
![extrafiles](https://user-images.githubusercontent.com/93156174/187292943-ca4839d9-6245-4b0c-bc75-cffa23771990.PNG)

**Comment by zippity21** [comment]
Oops 😳

Yeah, you can ignore those. I'll remove them from the release. Thank you.
